### PR TITLE
[RELENG-596] Add initial pinebuild support

### PR DESCRIPTION
**Pull request by escapewindow**
I basically searched for `devedition` and touched any relevant code.

### CODE REVIEW

File: beetmoverscript/docker.d/worker.yml [hunk 1]
```diff
@@ -17,6 +17,7 @@ bucket_config:
             key: { "$eval": "DEP_KEY" }
           buckets:
             devedition: 'net-mozaws-stage-delivery-archive'
+            pinebuild: 'net-mozaws-stage-delivery-archive'
```
Comment (jmaher) [inline beetmoverscript/docker.d/worker.yml]: all the other values are aligned, can you add an extra whitespace after `pinebuild:`

File: beetmoverscript/docker.d/worker.yml [hunk 2]
```diff
@@ -41,6 +42,7 @@ bucket_config:
             key: { "$eval": "NIGHTLY_KEY" }
           buckets:
             devedition: 'net-mozaws-prod-delivery-archive'
+            pinebuild: 'net-mozaws-prod-delivery-archive'
```
Comment (jmaher) [inline beetmoverscript/docker.d/worker.yml]: whitespace fix

File: beetmoverscript/docker.d/worker.yml [hunk 3]
```diff
@@ -50,6 +52,7 @@ bucket_config:
             key: { "$eval": "RELEASE_KEY" }
           buckets:
             devedition: 'net-mozaws-prod-delivery-archive'
+            pinebuild: 'net-mozaws-prod-delivery-archive'
```
Comment (jmaher) [inline beetmoverscript/docker.d/worker.yml]: whitespace fix

File: signingscript/src/signingscript/vendored/mozbuild/mozbuild/mozinfo.py [hunk 4]
```diff
@@ -75,6 +75,7 @@ def build_dict(config, env=os.environ):
     d['debug'] = substs.get('MOZ_DEBUG') == '1'
     d['nightly_build'] = substs.get('NIGHTLY_BUILD') == '1'
     d['release_or_beta'] = substs.get('RELEASE_OR_BETA') == '1'
+    # XXX pinebuild?
```
Comment (jmaher) [inline signingscript/src/signingscript/vendored/mozbuild/mozbuild/mozinfo.py:78]: can you file an issue for this?

**Review by jmaher — APPROVED** [review]
small nits- also are there docs to update?

**Comment by escapewindow** [comment]
> small nits- also are there docs to update?

I imagine not until pinebuild moves into releaseduty.

File: signingscript/src/signingscript/vendored/mozbuild/mozbuild/mozinfo.py (hunk 4 above)
Comment (escapewindow) [reply signingscript/src/signingscript/vendored/mozbuild/mozbuild/mozinfo.py:78]: https://mozilla-hub.atlassian.net/browse/RELENG-705

**Comment by MihaiTabara** [comment]
Having this skeleton should really help with the sprint items. Nice 👍

**Pull request merged**